### PR TITLE
28 bypassing crowdsec for local ips

### PR DESCRIPTION
**Pull request by mathieuHa**
Add bypass for local/vpn trusted IP

related to #28

### CODE REVIEW

File: bouncer.go [hunk 1]
```diff
@@ -82,6 +84,8 @@ type Bouncer struct {
 	defaultDecisionTimeout int64
 	customHeader           string
 	poolStrategy           *ip.PoolStrategy
+	poolStrategyTrusted    *ip.PoolStrategy
+	CheckerTrusted         *ip.Checker
```
Comment (maxlerebourg) [inline bouncer.go:88]: naming coherence

File: bouncer.go [hunk 2]
```diff
@@ -82,6 +84,8 @@ type Bouncer struct {
 	defaultDecisionTimeout int64
 	customHeader           string
 	poolStrategy           *ip.PoolStrategy
+	poolStrategyTrusted    *ip.PoolStrategy
```
Comment (maxlerebourg) [inline bouncer.go:87]: naming to close for the same thing

File: bouncer.go [hunk 3]
```diff
@@ -94,7 +98,16 @@ func New(ctx context.Context, next http.Handler, config *Config, name string) (h
...
-	checker, _ := ip.NewChecker(config.ForwardedHeadersTrustedIPs)
+	checker, err := ip.NewChecker(config.ForwardedHeadersTrustedIPs)
+	if err != nil {
+		logger.Debug("Checker == nil")
+		logger.Debug(err.Error())
+	}
+	checkerTrusted, err := ip.NewChecker(config.TrustedIPs)
+	if err != nil {
+		logger.Debug("CheckerTrusted == nil")
+		logger.Debug(err.Error())
```
Comment (maxlerebourg) [inline bouncer.go:109]: one if for 2 logs ?

File: exemples/trusted-ips/docker-compose.redis.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+version: "3.8"
```
Comment (maxlerebourg) [inline exemples/trusted-ips/docker-compose.redis.yml:1]: filename ?

File: bouncer.go (hunk 1 above)
Comment (mathieuHa) [reply bouncer.go]: fixed toghether

File: exemples/trusted-ips/docker-compose.redis.yml (hunk 4 above)
Comment (mathieuHa) [reply exemples/trusted-ips/docker-compose.redis.yml]: fixed

**Comment by mathieuHa** [comment]
Ready to MERGE

**Comment by rwjack** [comment]
Looks awesome! Deployed on dev servers right now

**Pull request merged**